ee/clickhouse/models/person.py

```diff
@@ -4,32 +4,55 @@
 from rest_framework import serializers
 
 from ee.clickhouse.client import async_execute, sync_execute
+from ee.clickhouse.models.clickhouse import generate_clickhouse_uuid
 from ee.clickhouse.sql.person import (
     DELETE_PERSON_BY_ID,
     GET_DISTINCT_IDS_SQL,
     GET_DISTINCT_IDS_SQL_BY_ID,
+    GET_PERSON_BY_DISTINCT_ID,
     GET_PERSON_SQL,
     INSERT_PERSON_DISTINCT_ID,
     INSERT_PERSON_SQL,
     PERSON_DISTINCT_ID_EXISTS_SQL,
     PERSON_EXISTS_SQL,
     UPDATE_PERSON_ATTACHED_DISTINCT_ID,
+    UPDATE_PERSON_IS_IDENTIFIED,
     UPDATE_PERSON_PROPERTIES,
 )
 from posthog.models.person import Person
 from posthog.models.team import Team
 
 
-def create_person(team_id: int, id: int, properties: Optional[Dict] = {}) -> int:
-    async_execute(INSERT_PERSON_SQL, {"id": id, "team_id": team_id, "properties": json.dumps(properties)})
-    return id
+def create_person(
+    team_id: int, distinct_ids: List[str], properties: Optional[Dict] = {}, sync: bool = False, **kwargs
+) -> int:
+    person_id = kwargs.get("person_id", None)  # type: Optional[str]
+    if not person_id:
+        person_id = generate_clickhouse_uuid()
 
+    if sync:
+        sync_execute(INSERT_PERSON_SQL, {"id": person_id, "team_id": team_id, "properties": json.dumps(properties)})
```

Comment:
This logic will all change over to this soon
https://github.com/PostHog/posthog/pull/1644/files#diff-5c7e83ef9d6e9c8db4c1e0f3d70c1557R26